index.html

```diff
@@ -46,6 +46,150 @@ <h3>LET'S CONNECT</h3>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="project-details" href="">See Project</a>
+                </div>
+            </section>
+
+            <section class="about">
+                <h1>About<br> Myself</h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h3 id="h3">LET'S CONNECT</h3>
+                <div class="contact">
+                    <ul id="social-media">
+                        <li class="github"><a href="#"><img src="/images/github.png" alt="github link"></a></li>
+                        <li class="linkedin"><a href="#"><img src="/images/Linkedin_icon.png" alt="linked link"></a>
+                        </li>
+                        <li class="doits"><a href="#"><img src="/images/doits.png" alt="doit"></a></li>
+                        <li class="twitter"><a href="#"><img src="/images/twitter.png" alt="twitter link"></a></li>
+                        <li class="m"><a href="#"><img src="/images/M.png" alt="M"></a></li>
+                    </ul>
+                    <a class="about-details" href="">Get My Resume</a>
+                </div>
+
+                <div class="about-skills">
+                    <div id="programming-lanuages">
+                        <div>
+                            <p>Languages</p>
+                            <img class = "Languages-img" src="images/fleche_dessous.png" alt="selected">
+                        </div>
+                        <ul>
+                            <li>
+                                <img class = "Languages-img" src="images/java.png" alt="javascript">
+                                <p>JavaSript</p>
+                            </li>
+                            <li>
+                                <img class = "Languages-img" src="images/html.png" alt="html">
+                                <p>HTML</p>
+                            </li>
+                            <li>
+                                <img class = "Languages-img" src="images/css.png" alt="css">
+                                <p>CSS</p>
+                            </li>
+                        </ul>
+                    </div>
+                    <div id="programming-frameworks">
+                        <p>Frameworks</p>
+                        <img class="framework-skills" src="images/select.png" alt="select">
+                    </div>
+                    <hr />
+                    <div id="skills">
+                        <p>Skills</p>
+                        <img class="framework-skills" src="images/select.png" alt="select">
+                    </div>
+                    <hr />
+                </div>
```

Comment:
- [ ] Your portfolio is looking great, well done! It would be even better if you used `ul` and `li` tags for technologies too as this is a better practice in HTML for listings, e.g:

```
<ul>
    <li>Languages
        <ul>
            <li>JS</li>
            <li>HTML</li>
            <li>CSS</li>
        </ul>
    </li>    
    <li>Frameworks</li>
    <li>Skills</li>
</ul>
```